src/handlers/thirdparty.ts

```diff
@@ -163,14 +162,18 @@ export function handleItemAdded(event: ItemAdded): void {
   let itemId = buildItemId(event.params._thirdPartyId, event.params._itemId)
 
   let item = Item.load(itemId)
-  if (item === null) {
+  if (item == null) {
     item = new Item(itemId)
   }
 
   item.blockchainItemId = event.params._itemId
   item.rawMetadata = event.params._metadata
   item.thirdParty = event.params._thirdPartyId
+  item.createdAt = event.block.timestamp
+  item.updatedAt = event.block.timestamp
+  item.reviewedAt = event.block.timestamp
```

Comment:
Should we have the same time for the `reviewedAt` and the `updatedAt` even if they weren't? Doesn't this overcomplicate the solution? That is, someone using the graph must know what if those values are equal, then it has not been updated nor reviewed, but that's not obvious.

Reply:
I'm following the implementation [here](https://github.com/decentraland/collections-graph/blob/master/src/handlers/collection.ts#L77) and more importantly [here](https://github.com/decentraland/builder/blob/master/src/modules/collection/utils.ts#L85) as we're trying to mimic collections. I'll have to use these values to fill up the TP collection so the front end works